### ISSUE TITLE
Remove alpha tag from Brand component

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#2713]
**Is your feature request related to a problem? Please describe.**
The Brand component is currently an alpha component.

This component should be amended and tested according to the a11y requirements to remove the tag.

**Describe the solution you'd like**
Remove alpha tag after a successful a11y swarm on the component.

**Describe alternatives you've considered**
 N/A

**Testing notes**
 [Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
Blocked on the Script Link a11y swarm and the integration of the Brand and Navigation in Simorgh locally.

Before removing the alpha tag we should confirm that https://github.com/bbc/simorgh/issues/4180#issuecomment-559372956 has been resolved
